.github/workflows/tests.yml

```diff
@@ -23,11 +23,12 @@ jobs:
       # --- Formatting checks ---
       - name: "Python formatting check"
         run: ./bin/check_black.sh
-      # Uncomment when all formatting done
       - name: "Run C/C++ formatting"
         run: ./bin/run_clang_format.sh
       - name: "Check C/C++ formatting changes"
         run: git diff --exit-code
+      - name: "Run C/C++ linter"
+        run: ./bin/run_clang_tidy.sh > /dev/null
```

Comment:
`clang-tidy` is _very_ verbose and difficult to read. This is because it inspects all header files that are compiled through the compilation database.

Discarding `stdout` leaves a leaner log, easier to inspect when something fails, as it just numbers the warnings generated and, if an error occurs, points to the file where it happened. Sufficient for CI imo.

Reply:
Yeah sounds good :+1:

Reply:
Yeah sounds good, should we put the `/dev/null` redirect in the script itself? Makes it nicer to run manually

Reply:
Indeed.